src/js/esse/JSONSchemasInterfaceClient.ts

```diff
@@ -0,0 +1,69 @@
+import { SchemaObject } from "ajv";
+import fs from "fs";
+import path from "path";
+
+import { walkDirSync } from "../scripts/utils";
+import { JSONInterfaceQuery } from "./types";
+
+export function readSchemaFolderSync(folderPath: string) {
+    const schemas: SchemaObject[] = [];
+
+    walkDirSync(folderPath, (filePath: string) => {
+        if (path.extname(filePath) !== ".json") {
+            return;
+        }
+        const schema = JSON.parse(fs.readFileSync(filePath).toString());
+        schemas.push(schema);
+    });
+
+    return schemas;
+}
+
+export default class JSONSchemasInterfaceClient {
```

Comment:
Functional:

1. Building and knowing which schemas we have (SERVER ONLY at the transpile time)

	Build Schemas
	Create Index (to know what's available)


2. Managing the in-memory set of schemas and returning schema to requester by ID

	Create in-memory cache with all schemas
	Expose the Index in memory

	Methods:

	- addSchema
	- getSchemaById

3. Extending the default list with additional schemas

	Add a set of schemas to in-memory cache by passing a folder